examples/README.md

```diff
@@ -2,19 +2,32 @@
 
 This directory contains short examples of using eqlpy to store and query encrypted records.
 
+Currently, there are 3 variations of supported ORMs and database driver combinations.
+
 ## Prerequisites
 
-### psycopg example
+These are the common prerequisites for all variations. See below for specific variations.
 
 * Python 3
 * Docker
 * Docker compose
 * CipherStash account
-* Supported database driver/ORM, either of:
-    * sqlalchemy + psycopg2
-    * psycopg 3
 * (Optinal) direnv or other tools to load environment variables from a file
 
+### psycopg example
+
+* psycopg 3
+
+### sqlalchemy example
+
+Please note currently pscyopg2 is required with the sqlalchemy example
+
+* sqlalchemy (2.0 or above recommended) + psycopg2
+
+### Django example
+
+* Django (5.1 or above recommended) + psycopg2
```

Comment:
I would like to see some more code examples here that give people an idea of what usage in Django looks like.

Reply:
It turns out that I forgot to add the entire example file 🤦  Now added and updated